custom/icds/tests/test_ls_no_submissions.py

```diff
@@ -0,0 +1,126 @@
+from datetime import date, datetime, timedelta
+from django.test import TestCase
+from mock import patch
+import pytz
+
+from corehq.apps.es.fake.users_fake import UserESFake
+from corehq.apps.domain.shortcuts import create_domain
+from corehq.apps.locations.tests.util import (
+    LocationStructure,
+    LocationTypeStructure,
+    setup_location_types_with_structure,
+    setup_locations_with_structure,
+)
+from corehq.apps.users.models import CommCareUser
+from custom.icds.messaging.indicators import LSSubmissionPerformanceIndicator
+
+
+@patch('corehq.apps.locations.dbaccessors.UserES', UserESFake)
+@patch('custom.icds.messaging.indicators.get_last_submission_time_for_users')
+class TestLSSubmissionPerformanceIndicator(TestCase):
+    domain = 'domain'
+
+    @classmethod
+    def setUpClass(cls):
+        super(TestLSSubmissionPerformanceIndicator, cls).setUpClass()
+        cls.domain_obj = create_domain(cls.domain)
+
+        location_type_structure = [
+            LocationTypeStructure('lsl', [
```

Comment:
the location type name for this is actually `supervisor`, i know it doesn't matter in this test but would be good to keep consistent i think